hyperscale/001-db-user-creation.sql

```diff
@@ -0,0 +1,3 @@
+CREATE USER citus WITH ENCRYPTED PASSWORD '###CITUS_PASSWORD###';
```

Comment:
thanks for figuring this out!

Reply:
@marcocitus  do you want me to remove 005-create_postgis_extension.sql or delete the content of the file?